dcos/mesos.py

```diff
@@ -177,7 +178,8 @@ def master_file_read(self, path, length, offset):
         url = self.master_url('files/read.json')
         params = {'path': path,
                   'length': length,
-                  'offset': offset}
+                  'offset': offset,
+                  'timeout': self._timeout}
```

Comment:
I think we shouldn't include this in `params`. Pass it as a regular function argument.